api/repos.go

```diff
@@ -81,6 +84,10 @@ func apiReposEdit(c *gin.Context) {
 		return
 	}
 
+	if !checkGroup(repo.LdapGroup) {
```

Comment:
Where is `checkGroup` defined? I can see `CheckGroup` (capital "C") only.

Reply:
cleaned all of that up